news/4126.bugfix

```diff
@@ -0,0 +1 @@
+Replace `pkg_resources` with `importlib.metadata`/`importlib.resources`/`packaging` @gforcada
```

Comment:
This gets added to `CHANGES.rst`, so we need to use reStructuredText markup for inline literals.

Note, we should change that to `CHANGES.md` in another PR, as I keep seeing this mistake from many contributors.

```suggestion
Replace ``pkg_resources`` with ``importlib.metadata``, ``importlib.resources``, or ``packaging``. @gforcada
```